radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/RadixHash.java

```diff
@@ -15,14 +16,35 @@
 public final class RadixHash {
 	private final byte[] hash;
 
-	private RadixHash(byte[] hash) {
-		this.hash = hash;
+	public RadixHash(byte[] hash) {
```

Comment:
Why public now? Should instead create static factory which copies byte array.

Reply:
Looks like this is only used for test. Please change test to use a mock instead.